test/pycheck/utils.py

```diff
@@ -552,13 +592,14 @@ def cleanup_users(self):
             self.sql(sql.SQL("DROP USER IF EXISTS {}").format(sql.Identifier(user)))
 
     def cleanup_schemas(self):
-        for dbname, schema in self.schemas:
-            self.sql(
-                sql.SQL("DROP SCHEMA IF EXISTS {} CASCADE").format(
-                    sql.Identifier(schema)
-                ),
-                dbname=dbname,
-            )
+        with self.silent_logs():
```

Comment:
What errors/warnings is this silencing? Would be good to add a comment, because I wouldn't expect the below code to throw an error/warning.